jest.config.js

```diff
@@ -134,6 +134,7 @@ module.exports = {
         }],
         ["github-actions", { "silent": false } ]
     ],
+    "testResultsProcessor": "jest-sonar-reporter",
```

Comment:
Should the "jest-sonar-reporter" be added to the reporters list above?

I'm kind of worried about the `testResultsProcessor` property, since it had a tendency to override some reporters (probably in older versions of jest). IDK if you remember, but that's why Dan's `jest-stare` reporter had a list of `additionalReporters` where we had the `jest-junit` and `jest-sonar`.

If this is not something to be concerned about in newer versions of Jest, then I'm fine with leaving it as-is.

Reply:
I tried that and it doesn't work. jest-sonar-reporter does not support the newer interface, and has been archived since 2019. Nothing else seems to exist outside of beta for providing Jest results to Sonar.